components/admin_console/admin_definition.jsx

```diff
@@ -4904,7 +4903,7 @@ const AdminDefinition = {
                                 value: Constants.GOOGLE_SERVICE,
                                 display_name: t('admin.oauth.google'),
                                 display_name_default: 'Google Apps',
-                                isHidden: it.all(it.not(it.licensedForFeature('GoogleOAuth')), it.not(it.cloudLicensed)),
+                                isHidden: it.not(it.licensedForFeature('GoogleOAuth')),
```

Comment:
Shouldn't google be enabled for cloud free?

Reply:
@BenCookie95 apologies for I didn't notify you, the revert PRs will most likely be closed. I hope you didn't waste too much of your time.

Thanks for reviewing still!